closure/goog/editor/plugins/enterhandler.js

```diff
@@ -757,6 +781,21 @@ goog.editor.plugins.EnterHandler.isInOneContainerW3c_ = function(range) {
 goog.editor.plugins.EnterHandler.isPartialEndW3c_ = function(range) {
   var endContainer = range.getEndNode();
   var endOffset = range.getEndOffset();
+  // Since the range object is different for each browser,
+  // Normalize range using previousSibling or parentNode of
+  // endContainer, when endOffset is 0.
+  while (endOffset == 0 && endContainer) {
+    if (endContainer.previousSibling) {
+      endContainer = endContainer.previousSibling;
+      endOffset = goog.editor.node.getLength(endContainer);
+    } else if (endContainer.parentNode) {
+      endContainer = endContainer.parentNode;
+      endOffset = 0;
+    } else {
+      break;
+    }
+  }
```

Comment:
Could this be replaced with a call to "goog.editor.range.normalize"?

Reply:
OK! I'll check it.

Reply:
It seems can replace with a call to "goog.editor.range.normalize".
but, The result wasn't to be the expected return value...

Reply:
That's fine. I'll take a closer look at this internally and change it if there's something similar we can use instead.

Reply:
Thanks！！